### PR TITLE
update the fields and search to use the folded search field

### PR DESCRIPTION
**Pull request by jonhartzler**
A couple of changes.

Change the search to have a filtered field that uses the ascii folded filter which removes diacritics from the text and the search for comparison. 

Changed the id in elastic search. 

Changed how we identify fields in the highlights.   The new field name has a dot in it so I had to use strings and []

### CODE REVIEW

File: @ndlib/gatsby-theme-marble/src/components/Shared/ManifestCard/ManifestCardChildren/index.js [hunk 1]
```diff
@@ -55,8 +55,8 @@ export const findMetadata = (manifest, options) => {
 }
 
 export const findCreator = (item, parentProps) => {
-  if (parentProps.highlight && parentProps.highlight.creator) {
-    return parentProps.highlight.creator[0]
+  if (parentProps.highlight && parentProps.highlight['creator.folded']) {
```
Comment (danthewolfe) [inline @ndlib/gatsby-theme-marble/src/components/Shared/ManifestCard/ManifestCardChildren/index.js:58]: Should we just make this a check with `typy`? 
```if (typy(parentProps, 'highlight.creator.folded').isArray) {```

File: @ndlib/gatsby-theme-marble/src/components/Shared/SearchTools/SearchResults/HitDisplay/test.js [hunk 2]
```diff
@@ -24,3 +24,16 @@ test('HitDisplay', () => {
...
+  expect(wrapper.find(DisplayViewToggle).exists()).toBeTruthy()
+  expect(wrapper.find(ManifestCard).at(0).children().html()).toEqual('<div class="css-1itje8o"><em>data</em></div>')
+  expect(wrapper.find(ManifestCard).at(2).children().html()).toEqual('<div class="css-1itje8o"><em>line2data</em></div>')
+})
+
+test('all metadata hightlights splits on :: and shows only the line with the data', () => {
+  const wrapper = shallow(<HitDisplay hits={hits} />)
+  expect(wrapper.find(DisplayViewToggle).exists()).toBeTruthy()
+  expect(wrapper.find(ManifestCard).at(2).children().html()).toEqual('<div class="css-1itje8o"><em>line2data</em></div>')
+})
```
Comment (danthewolfe) [inline @ndlib/gatsby-theme-marble/src/components/Shared/SearchTools/SearchResults/HitDisplay/test.js:39]: Do we need an additional test when there are no hits (default empty search scenario)?

**Review by danthewolfe — APPROVED** [review]
See comments, but 👍

File: @ndlib/gatsby-theme-marble/src/components/Shared/ManifestCard/ManifestCardChildren/index.js (hunk 1 above)
Comment (jonhartzler) [reply @ndlib/gatsby-theme-marble/src/components/Shared/ManifestCard/ManifestCardChildren/index.js:58]: let me try I was not sure about how typy was going to handle that being a property not an object.

File: @ndlib/gatsby-theme-marble/src/components/Shared/SearchTools/SearchResults/HitDisplay/test.js (hunk 2 above)
Comment (jonhartzler) [reply @ndlib/gatsby-theme-marble/src/components/Shared/SearchTools/SearchResults/HitDisplay/test.js:39]: yes

File: @ndlib/gatsby-theme-marble/src/components/Shared/ManifestCard/ManifestCardChildren/index.js (hunk 1 above)
Comment (jonhartzler) [reply @ndlib/gatsby-theme-marble/src/components/Shared/ManifestCard/ManifestCardChildren/index.js:58]: this is not working.  it does not seem to id keys with dots in them.
